appservice/src/intTest/groovy/nu/yona/server/CreateUserOnBuddyRequestTest.groovy

```diff
@@ -47,6 +47,25 @@ class CreateUserOnBuddyRequestTest extends AbstractAppServiceIntegrationTest
 		// TODO: How to delete the invited user?
 	}
 
+	def 'Bob opens the link and goes to the buddy invitation deep link landing page'()
+	{
+		given:
+		def richard = addRichard()
+		def mobileNumberBob = "+$timestamp"
+		def deepLinkLandingPageURL = buildInviteDeepLinkLandingPageUrl(sendBuddyRequestForBob(richard, mobileNumberBob))
```

Comment:
This is a bit a strange test. The actor here is Richard, but he's masquerading Bob. Please add an explanatory comment.